src/data-tutorials/users.tsx

```diff
@@ -1626,6 +1626,13 @@ const Users: User[] = [
     website: 'blog/tutorials/build-a-generic-staking-dapp-using-foundry-and-nextjs',
     tags: ['celosage', 'advanced', 'smartcontract', 'solidity', 'nextjs', 'materialui', 'react'],
   },
+  {
+    title: 'Connect and interact with Celo using Web3React',
+    description: 'Alternative method of connecting to Celo using web3React',
+    preview: require('./showcase/advanced/connect-and-interact-with-celo-using-web3react.png'),
+    website: 'blog/tutorials/connect-and-interact-with-celo-using-web3react',
+    tags: ['celosage', 'advanced', 'smartcontract', 'solidity', 'nextjs', 'materialui', 'react'],
```

Comment:
Hey make sure the tags are same in metadata of .md file and also the users.tsx file